packages/docusaurus-theme-common/src/hooks/useCodeWordWrap.ts

```diff
@@ -5,16 +5,15 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import type {
-  MutableRefObject} from 'react';
-import {
-  useState,
-  useCallback,
-  useEffect,
-  useRef
-} from 'react';
-
-export function useCodeWordWrap() {
+import type {MutableRefObject} from 'react';
+import {useState, useCallback, useEffect, useRef} from 'react';
+
+export function useCodeWordWrap(): {
+  readonly codeBlockRef: (node: HTMLPreElement | null) => void;
```

Comment:
this is not needed to use a function here you can return the React ref and assign it directly (like it was before, but instead it's created in the custom hook instead of the component